src/regexp.toit

```diff
@@ -1271,30 +1271,34 @@ class Capture_ extends MiniExpAst_:
 interface Match:
   /// Returns the regular expression that created this match.
   pattern -> RegExp
-  /// Returns the input string that was matched by the regexp.
+
+  /// Returns the input string in which a match was found.
   input -> string
+
+  /// Returns the substring of the input that matched the regexp.
+  matched -> string
```

Comment:
Why not just `match` ?

Reply:
Because the whole class is called Match, and it's strange to have a method in it that is called match, but doesn't return a Match.